Can't install via npm: HTTP 406 from github for defunctzombie/stacktrace.js#07e7b95

When trying to install zuul via npm, I'm getting an HTTP 406 from github for the dependency ``"defunctzombie/stacktrace.js#07e7b95"``.  At first I thought this was a symptom of github's DDoS, but it looks like this arises from not having a fully-qualified URL with "git://" protocol in the package.json.

In short, npm is trying to resolve this dependency as ``https://github.com/defunctzombie/stacktrace.js#07e7b95``; github now returns a 406 for this; but it would work fine to instead use ``git://github.com/defunctzombie/stacktrace.js#07e7b95``.

It looks like npm will be [working around this](https://github.com/npm/npm/issues/7630) in a future release, but it'd be good to fix it here so non-bleeding-edge npm users can still install zuul.

Is there any solution?

```
$ npm i -g zuul
npm ERR! fetch failed https://github.com/defunctzombie/stacktrace.js#07e7b95
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 406
```

--- a/HISTORY.md
+++ b/HISTORY.md
@@ -1,6 +1,7 @@
 # UNRELEASED
 
 * Builder API change #182
+* Specify http tarball url for stacktracejs to be compatible with node 0.10 npms #188
 
 # 2.1.1 (2015-03-09)
 

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "shallow-copy": "0.0.1",
     "shell-quote": "1.4.1",
     "stack-mapper": "0.2.2",
-    "stacktrace-js": "defunctzombie/stacktrace.js#07e7b95",
+    "stacktrace-js": "http://github.com/defunctzombie/stacktrace.js/tarball/07e7b9516f1449f5c209e4f67f11a43f738c1712",
     "superagent": "0.15.7",
     "superstack": "0.0.4",
     "tap-finished": "0.0.1",

